tests/training/features/factors/test_factors.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+
+#####################################################################
+# SUMMARY: Training a factored model 
+# AUTHOR: pedrodiascoelho
+# TAGS: factors
+#####################################################################
+
+# Exit on error
+set -e
+
+# Remove old artifacts and create working directory
+rm -rf factors factors.{log,out,diff}
+mkdir -p factors
+
+# Run marian command
+$MRT_MARIAN/marian \
```

Comment:
Please add `--clip-norm 0` to the test or explicitly state `--clip-norm 1` if clip-norm 0 is not stable. `--clip-norm 0` is currently buggy and either it will change (fixed and/or disabled by default).